src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Win32.cs

```diff
@@ -257,58 +254,53 @@ internal static ProcessInfo[] GetProcessInfos(Predicate<int>? processIdFilter =
             ProcessInfo[] processInfos;
 
             // Start with the default buffer size.
-            int bufferSize = DefaultCachedBufferSize;
+            int bufferSize = DefaultCachedBufferSize * sizeof(long);
```

Comment:
Perhaps change the constants to remove `long` altogether?

Reply:
good idea, done